### PR TITLE
Added MessageChannel#editMessageById

### PR DESCRIPTION
**Pull request by Asourcious**

### CODE REVIEW

File: src/main/java/net/dv8tion/jda/core/entities/MessageChannel.java [hunk 1]
```diff
@@ -242,4 +247,38 @@
...
     RestAction<List<Message>> getPinnedMessages();
+
+    default RestAction<Message> editMessageById(String id, String newContent)
+    {
+        return editMessageById(id, new MessageBuilder().appendString(newContent).build());
+    }
+
+    default RestAction<Message> editMessageById(String id, Message newContent)
+    {
+        JSONObject json = new JSONObject().put("content", newContent.getRawContent()).put("tts", newContent.isTTS());
```
Comment (DV8FromTheWorld) [inline src/main/java/net/dv8tion/jda/core/entities/MessageChannel.java]: There need to be Args.notNull checks on both id and Message.

Comment (DV8FromTheWorld) [inline src/main/java/net/dv8tion/jda/core/entities/MessageChannel.java]: MessageBuilder does that already. Let the MessageBuilder catch that error and filter back.

**Comment by Asourcious** [comment]
Done

**Comment by DV8FromTheWorld** [comment]
So uh, this kinda got borked due to the implementation of MessageImpl#toJSONObject.

If you could re-PR with the changes that'd be great. Sorry about that.

**Pull request closed without merging**